src/egit.c

```diff
@@ -121,6 +122,9 @@ static void egit_finalize(void* _obj)
         repo = git_reference_owner(obj->ptr);
         git_reference_free(obj->ptr);
         break;
+    case EGIT_BLAME:
+        git_blame_free(obj->ptr);
```

Comment:
The `git_blame` struct holds a repository pointer. Do we need to keep that alive? I'm not sure, since there aren't any getters for it the way there are for references and objects.